docs/important/relevant_functionalities_de.md

```diff
@@ -0,0 +1,1250 @@
+    {
+      "Name": "service-account-remove",
+      "Description": "Removes a service account",
+      "Command": "/remove-sa.sh"
+    }
+  ],
+  ...,
+}
+```
+
+Die Begriffe `service-account-create` und `service-account-remove` sind geschützte Eigenschaften und werden von Clients, die die `dogu.json` interpretieren als Befehle zum Erstellen und Löschen von Service Accounts benutzt.
+
+In den beiden folgenden Abschnitten wird auf die Erzeugung und Löschung von Service Accounts eingegangen. Weitere Information über ExposedCommands befinden sich im [ExposedCommands](../core/compendium_de.md#exposedcommands)-Abschnitt des Kompendiums.
+
+#### Service Account anlegen
+
+Das `service-account-create`-Skript verwaltet das Erstellen eines neuen Service Accounts. Es wird immer ausgeführt, wenn ein neuer Konsument installiert wird. Während des Aufrufs erhält das `service-account-create`-Skript jedenfalls den Namen des Konsumenten-Dogus als erstes Argument. Ein zweites Argument zur Feineinstellung des Service Accounts ist optional und hängt von der Unterstützung des Produzenten-Dogus ab.
+
+Ein `service-account-create`-Skript besteht aus drei Schritten:
+
+1. Erstellen der Account Informationen
+2. Anlegen eines Accounts für unsere Software mit den Informationen
+3. Schreibe die Accountinformationen verschlüsselt in den Dogu-spezifischen Konfigurationsbereich des Verbrauchers.
+
+Ein Beispiel für das `service-account-create`-Skript könnte folgendermaßen aussehen:
+
+`create-sa.sh`
+
+```bash
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+doguName="${1}"
+{
+  #1) Erstellen der Zugangsdaten
+  USER="${doguName}-$(doguctl random)"
+  PASSWORD=$(doguctl random)
+
+  #2) Legt einen Account für unsere Software mit den Informationen an
+  # ... Implementiere die Dogu-spezifische Logik hier
+} >/dev/null 2>&1 # unterdrücke sonstige Ausgaben auf stdout, da nur Zugangsdaten oder Fehler erwartet werden
+
+  #3) Gib Zugangsdaten zurück. Der Client schreibt die Zugangsdaten verschlüsselt in den Dogu-spezifischen Konfigurationsbereich des Verbrauchers.
+echo "username: ${USER}"
+echo "password: ${PASSWORD}"
+```
+
+In Schritt 1 werden Zugangsdaten zufällig generiert. Der Name des Konsumenten-Dogus kann in die Zugangsdaten eingearbeitet werden, allerdings ist dies keine Pflicht.
+
+In Schritt 2 muss ein Account mit den generierten Informationen in der Dogu-Software erstellt werden. 
+
+In Schritt 3 werden die Zugangsdaten mit dem Format `echo "<registrykey>: <registryvalue>"` ausgegeben. `registrykey` und `registryvalue` müssen durch einen Doppelpunkt und einem Leerzeichen getrennt sein. Die gesamte Zeile muss durch ein Newline beendet werden.
+
+Diese Zugangsdaten werden vom verarbeitenden Client automatisch eingelesen und in den Dogu-spezifischen Konfigurationsbereich des Verbrauchers verschlüsselt abgelegt.
+Nach dem obigen Beispiel würde dem Konsumenten-Dogu zwei Registry-Einträge einrichten, die das jeweilige Zugangsdatum als Wert enthalten:  
```

Comment:
```suggestion
Nach dem obigen Beispiel würde das Konsumenten-Dogu zwei Registry-Einträge einrichten, die das jeweilige Zugangsdatum als Wert enthalten:  
```

Reply:
Better for the english translation as this could easily be overseen as 'access date'.
```suggestion
Nach dem obigen Beispiel würde das Konsumenten-Dogu je einen Registry-Eintrag für Username und Passwort erzeugen:  
```

Reply:
Done. I chose a different word

Reply:
argh, I changed it again :D
See 24488a4c776273cc16b6228eae2dfdc714dcd198